### ISSUE TITLE
reproduce your results

### ISSUE THREAD

**Issue by zzksdu** [VITA-MLLM/VITA#106]
As introduced in your paper, the VITA training process consists of 3 steps. 
 The first step is to finetune llm module. 
 The second step is multimodel alignment 
 The third part is multimodel instruction tuning.

In your code, there are many training scripts. Can you indicate which script corresponds to which step of training?

Another question is, in your source code, the language model uses qwen2, so is the final language model qwen2 or mixtral 8 * 7B?
@wangxiongts @BradyFU @linhaojia13 @longzw1997

**Comment by zzksdu** [comment]
Do you have any plans to make your training dataset public?

**Comment by linhaojia13** [comment]
VITA-1.0 uses Mixtral as its base language model, while VITA-1.5 uses Qwen2.5-7B-Instruct. Currently, VITA-1.0 is deprecated, so let me explain the training stages for VITA-1.5:
- `pretrain_mlp_qwen_nodes.sh`: Stage 1.1
- `finetune_qwen_nodes.sh`: Stage 1.2
- `finetuneTask_qwen_nodes.sh`: Stage 1.3
- `finetuneTaskNeg_qwen_nodes.sh`: Stage 2.2

As for the datasets used, they are not publicly available, but the majority of them consist of open-source data.

**Comment by zzksdu** [comment]
@linhaojia13  在论文中， s1.3是 unfeeze vision tower + mlp + llm， 为什么在finetuneTask_qwen_nodes.sh与finetune_qwen_nodes.sh不太相同，finetune_qwen_nodes.sh中添加了unfreeze_vision_tower，而s1.3的脚本中没有这个参数。

![Image](https://github.com/user-attachments/assets/7c6244e9-23ef-4050-9b37-1f71daf6a715)
另外一个问题是： 
按照你stage1.3配置， 会出现这个UserWarning: None of the inputs have requires_grad=True. Gradients will be None，意味着没有参数在更新。
